packages/odyssey-storybook/src/components/odyssey-mui/MenuButton/MenuButton.stories.tsx

```diff
@@ -0,0 +1,138 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Story } from "@storybook/react";
+import {
+  Divider,
+  ListItemIcon,
+  ListItemText,
+  ListSubheader,
+  MenuButton,
+  MenuItem,
+  UserGroupIcon,
+  GlobeIcon,
+  CalendarIcon,
+  OverflowVerticalIcon,
+} from "@okta/odyssey-react-mui";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+
+import MenuButtonMdx from "./MenuButton.mdx";
+
+export default {
+  title: `MUI Components/Menu Button`,
+  component: MenuButton,
+  parameters: {
+    docs: {
+      page: MenuButtonMdx,
+    },
+  },
+  argTypes: {
+    children: {
+      control: "text",
+      defaultValue: undefined,
+    },
+    buttonLabel: {
+      control: "text",
+      defaultValue: "More actions",
+    },
+    buttonEndIcon: {
+      control: "text",
+      defaultValue: undefined,
+    },
+    buttonVariant: {
+      control: "text",
+      defaultValue: undefined,
+    },
+  },
+  decorators: [MuiThemeDecorator],
+};
+
+const DefaultTemplate: Story = (args) => {
+  return (
+    <MenuButton
+      buttonEndIcon={args.buttonEndIcon}
+      buttonLabel={args.buttonLabel}
+      buttonVariant={args.buttonVariant}
+    >
```

Comment:
We can do `<MenuButton {...args} />` after we assign the TS to it. I think that works. You'll wanna check the "Show Code" after making the change to see.

Reply:
Would you mind if we kept these explicit? Since the stories are docs, I like being able to see all of the prop application.

Reply:
What does "Show Code" show in either instance?

I tried going to https://de7142a.ods.dev/, but it doesn't exist.

Reply:
<img width="254" alt="Screenshot 2023-02-16 at 12 33 44 PM" src="https://user-images.githubusercontent.com/36284167/219480774-be126a55-ad1e-46cb-b527-de50e3089e41.png">

They're identical; however, I meant less in the rendered docs and more in the code itself. Not strongly opinionated here; I just like being explicit 😁

Reply:
I'd prefer using `{...args}` then if "Show Code" is the same for both. That way I don't have to update the code in multiple places when adding or removing props.